lib/interfaces.d.ts

```diff
@@ -4,6 +4,12 @@ export type Url = string;
 
 export type PhoneNumber = string;
 
+export type PhoneNumberCapabilities = {
+  mms: boolean;
+  sms: boolean;
+  voice: boolean;
```

Comment:
What about fax?

Reply:
I did not see that here:
https://code.hq.twilio.com/twilio/api-definitions/blob/master/transactions/api/v2010/incoming_phone_number.json#L42

Where did you see `fax` defined?